### PR TITLE
feat: support kafka 2.8.2 with zookeeper

### PR DESCRIPTION
**Pull request by skyrise-l**
https://github.com/apecloud/kubeblocks-addons/issues/890
Support for Kafka version 2.8.1, using Zookeeper. Added a new mode: withZookeeper. When enabled, it uses both Kafka-2.8.1 and Zookeeper components.

### CODE REVIEW

File: addons/kafka/scripts/kafka-server-zk-setup.sh.tpl [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  # Todo: enable encrypted transmission inside the service
+  #export KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP=CONTROLLER:SSL,INTERNAL:SSL,CLIENT:SSL
+  #export KAFKA_CFG_SECURITY_INTER_BROKER_PROTOCOL=SSL
+  #echo "KAFKA_CFG_SECURITY_INTER_BROKER_PROTOCOL=SSL"
+
+  mkdir -p /opt/bitnami/kafka/config/certs
+  PEM_CA="$KB_TLS_CERT_PATH/ca.crt"
+  PEM_CERT="$KB_TLS_CERT_PATH/tls.crt"
+  PEM_KEY="$KB_TLS_CERT_PATH/tls.key"
+  if [[ -f "$PEM_CERT" ]] && [[ -f "$PEM_KEY" ]]; then
```
Comment (nayutah) [inline addons/kafka/scripts/kafka-server-zk-setup.sh.tpl:27]: also check the PEM_CA ?

File: addons/kafka/templates/cmpd-broker-2.8.1.yaml [hunk 2]
```diff
@@ -0,0 +1,201 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: ComponentDefinition
+metadata:
+  name: "{{ include "kafka-broker.componentDefName" . }}-2.8.1"
```
Comment (nayutah) [inline addons/kafka/templates/cmpd-broker-2.8.1.yaml:4]: refact all version string to 2.8.2

Comment (skyrise-l) [reply addons/kafka/templates/cmpd-broker-2.8.1.yaml:4]: The 2.8.2 image is still being tested. I'll push it again once there are no issues.

**Comment by nayutah** [comment]
resolve the conflicts

**Comment by skyrise-l** [comment]
> resolve the conflicts

it  resolved

**Review by nayutah — APPROVED** [review]

**Pull request closed without merging**